.rubocop.yml

```diff
@@ -126,3 +130,6 @@ Style/MethodCalledOnDoEndBlock:
 # value in using "" for all strings irrespective of whether string interpolation is used
 Style/StringLiterals:
   EnforcedStyle: double_quotes
+
+Style/HashSyntax:
+  Enabled: false
```

Comment:
I think this should be enabled in the master style within the gem and not overruled here.

Reply:
ah yeah, it is already! I can remove this. thanks